nomad/resources_endpoint.go

```diff
@@ -73,6 +73,15 @@ func getResourceIter(context, prefix string, ws memdb.WatchSet, state *state.Sta
 	}
 }
 
+// If the length of a string is odd, return a subset of the string to the last
+// even character
+func roundDownIfOdd(s string) string {
```

Comment:
Ok, understood. So we can round the UUID for the query, and then use a filter function for the remaining character.